client/state/reader/teams/actions.js

```diff
@@ -7,11 +7,26 @@
  */
 import {
 	READER_TEAMS_REQUEST,
+	READER_TEAMS_RECEIVE,
 } from 'state/action-types';
 
 export function requestTeams() {
 	return {
 		type: READER_TEAMS_REQUEST,
+		meta: {
+			requestStart: READER_TEAMS_REQUEST
+		}
+	};
+}
```

Comment:
The thing that bugs me a bit here is that we're leaking knowledge. I'm not sure who really needs to know that we need to dedupe requests, outside of the data middleware. It's not a terrible leakage, as the requestKey could be used for other things I suppose. 

I'm not sure I love `requestStart` as the kickoff. Yes, I know I suggested it. :) 

From the action creator's standpoint, something like `keyedRequest( baseAction )` would probably make more sense? Then the data-layer would dispatch something like `completeKeyedRequest( baseAction )`